src/App.jsx

```diff
@@ -1,22 +1,35 @@
-import './App.css'
-import TodoList from './TodoList'
-import AddTodoForm from './AddTodoForm'
-import { useState } from 'react'
+import "./App.css";
+import TodoList from "./TodoList";
+import AddTodoForm from "./AddTodoForm";
+import { useEffect, useState } from "react";
+
+const savedTodoList = localStorage.getItem("savedTodoList");
```

Comment:
Move this and `initialList` inside the `useSemiPersistentState` hook for good hygiene (since these are only used in there)

Reply:
Thanks for the tip! I will put the changes on the next assignment PR. :)